tests/bgp-extended-msg.out

```diff
@@ -0,0 +1,30 @@
+    1  19:15:31.105336 IP (tos 0xc0, ttl 1, id 52866, offset 0, flags [DF], proto TCP (6), length 147)
+    192.168.10.68.179 > 192.168.10.252.35857: Flags [P.], cksum 0xe752 (correct), seq 3505597008:3505597103, ack 4282450256, win 227, options [nop,nop,TS val 124114 ecr 2546325983], length 95: BGP
+	Open Message (1), length: 95
+	  Version 4, my AS 65001, Holdtime 180s, ID 192.168.10.68
+	  Optional parameters, length: 66
+	    Option Capabilities Advertisement (2), length: 6
+	      Multiprotocol Extensions (1), length: 4
+		AFI IPv4 (1), SAFI Unicast (1)
+	    Option Capabilities Advertisement (2), length: 2
+	      Route Refresh (Cisco) (128), length: 0
+	    Option Capabilities Advertisement (2), length: 2
+	      Route Refresh (2), length: 0
+	    Option Capabilities Advertisement (2), length: 2
+	      Enhanced Route Refresh (70), length: 0
+		no decoder for Capability 70
+	    Option Capabilities Advertisement (2), length: 6
+	      32-Bit AS Number (65), length: 4
+		 4 Byte AS 65001
+	    Option Capabilities Advertisement (2), length: 2
+	      BGP Extended Message (6), length: 0
+	    Option Capabilities Advertisement (2), length: 6
+	      Multiple Paths (69), length: 4
+		AFI IPv4 (1), SAFI Unicast (1), Send/Receive: Receive
+	    Option Capabilities Advertisement (2), length: 18
+	      Unknown (73), length: 16
+		no decoder for Capability 73
```

Comment:
Why this Capability 73 (unkown) is present?
Seems to be linked to an expired draft?

Reply:
It's because https://github.com/the-tcpdump-group/tcpdump/pull/939 was rejected :)